.travis.yml

```diff
@@ -1,10 +1,11 @@
 language: python
 
-python:
-  - 2.7
-  - 3.5
 matrix:
  include:
+  - python: "2.7"
+    env: DISTRIBUTIONS="sdist bdist_wheel"
+  - python: "3.5"
+    env: DISTRIBUTIONS="bdist_wheel"
```

Comment:
Is there a reason why this can't be a universal wheel that works with all Python versions?  I wasn't aware of any C extensions in this package.

Reply:
Good point! I've updated it to be a universal wheel.